[Visual Studio] (TypeScript) Removing obsolete VS conversion steps (#1733)
OfficeDev/office-js-docs-pr: docs/develop/convert-javascript-to-typescript.md

4. From the **Tools** tab, choose **NuGet Package Manager** and then select **Manage NuGet Packages for Solution...**.

> 5. In the same search box, enter **jquery.TypeScript.DefinitelyTyped**. Install or update this package if it is already installed. This will add the jQuery TypeScript definitions into your project. The packages for both jQuery and Office.js will now appear in a new file generated by Visual Studio, called **packages.config**.

Reviewer: ```suggestion
5. With the **Browse** tab selected, enter **jquery.TypeScript.DefinitelyTyped**. Install this package, or update it if it's already installed. This will ensure the jQuery TypeScript definitions are included in your project. The packages for jQuery appear in a file generated by Visual Studio, called **packages.config**.
```

            var element = document.querySelector('.MessageBanner');
            messageBanner = new components.MessageBanner(element);
            messageBanner.hideBanner();

>             // If not using Excel 2016, use fallback logic.

Reviewer: Can you clarify the Excel versions here? Is this condition applicable to Excel 2013 or Excel 2019?

```suggestion
            // If not using Excel 2016 or later, use fallback logic.
```

Author: Good call. I flipped it to explicitly call out the usage of Excel 2013. I think it's more clear to point to 2013 as the outlier.

            var element = document.querySelector('.MessageBanner');
            messageBanner = new components.MessageBanner(element);
            messageBanner.hideBanner();

            // If you're using Excel 2013, use fallback logic.

>             if (!Office.context.requirements.isSetSupported('ExcelApi', 1.1)) {

Reviewer: Shouldn't this be returned to the string version?
```suggestion
            if (!Office.context.requirements.isSetSupported('ExcelApi', '1.1')) {
```
